portfolio/src/main/webapp/index.html

```diff
@@ -8,21 +8,27 @@
   </head>
   <body>
     <div id="pageWrapper">
+      <div id="navbar">
+        <div class="navButton" onClick="highlightSection('basicInfo')">Introduction</div> 
+        <div class="navButton" onClick="highlightSection('hobby-wrapper')">Hobbies</div> 
```

Comment:
I noticed that sometimes you're using camelCase and sometimes you're using dash-separated-string. Can we be consistent? I think camelCase is more common, so maybe we can use that.

Reply:
Done; changed naming to camelCase.